internal/gl/gl_js.go

```diff
@@ -351,6 +357,12 @@ func (f *Functions) LinkProgram(p Program) {
 	f.Ctx.Call("linkProgram", js.Value(p))
 }
 func (f *Functions) PixelStorei(pname Enum, param int) {
+	if !f.isWebGL2 {
+		switch pname {
+		case PACK_ROW_LENGTH, UNPACK_ROW_LENGTH:
+			return // PACK_ROW_LENGTH and UNPACK_ROW_LENGTH is only available on WebGL 2
```

Comment:
We can't just ignore the PixelStorei call. Panic?

Reply:
Technically, the `PixelStorei` is supported on WebGL 1. I don't knwo if `PixelStorei` is called with other  types of `pname`. I'll test it later. :)

Reply:
Both PACK_ROW_LENGTH and UNPACK_ROW_LENGTH are unsupported in webgl 1 according to https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/pixelStorei. Not sure why PACK... worked until now. In any case, you can probably get away with skipping the pixelstorei call if the row length is equal to the image stride.